### ISSUE TITLE
release to forge

### ISSUE THREAD

**Issue by craiggenner** [meltwater/puppet-incron#17]
Please can you release a new version to the puppet forge.

The forge version is missing a few fixes that are in the git repo.

**Comment by marcofl** [comment]
yes, please release to forge. or at last push a tag to git. thank you very much.
